examples/uma-server/vasp2.go

```diff
@@ -363,7 +363,10 @@ func (v *Vasp2) handleUmaPayreq(context *gin.Context) {
 		ExpirySecs:       &expirySecs,
 	}
 
-	conversionRate := int64(34_150)
+	conversionRate := 1000.0
+	if request.CurrencyCode == "USD" {
+		conversionRate = 22883.56
```

Comment:
Maybe make this a constant somewhere? `const MillisatoshiPerUsd = 22883.56`

Reply:
Done :-)